packages/web-components/src/components/card/card.ts

```diff
@@ -140,6 +177,12 @@ class DDSCard extends StableSelectorMixin(DDSLink) {
   @property()
   href = '';
 
+  /**
+   * Pictogram position
+   */
+  @property({ attribute: 'pictogram-position', reflect: true })
+  pictogramPosition = 'top';
```

Comment:
Can we introduce an enum and use it here? A note is that we recently moved all enum definitions to `src/components/{{name-of-component}}/defs.ts`. Please refer to how `<dds-link-list>` defines its enum.